docs/extend/extend-apiml/api-medation-sso-integration-extenders.md

```diff
@@ -11,15 +11,17 @@ This article does not cover the client methods to call API ML and authenticate.
 
 To integrate with API Mediation Layer and leverage Single Sign On, choose from the following three possible methods:
 
-- Accepting JWT token (recommended)
-- Accepting SAF IDT token
-- Accepting PassTicket
+- [Accepting JWT token (recommended)](#accepting-jwt)
+- [Accepting SAF IDT token](#accepting-saf-idt)
+- [Accepting PassTicket](#accepting-passtickets)
 
-Two additional possibilities can potentially be leveraged to enable Single Sign On but are not properly integrated with the standard API ML:
+Additional possibilities can potentially be leveraged to enable Single Sign On but are **not** properly integrated with the standard API ML:
 
-- Bypassing the authentication for the service
-- Asking for details about the x509 certificate used for authentication  
-**Note:** Asking for details about the x509 certificate does not properly participate in SSO as this method cannot accept all authentication methods that are supported upstream of API Mediation Layer. 
+* [Bypassing authentication for the service](#bypassing-authentication-for-the-service)   
+    **Note:** This option is for SSO only if the service does not have an authenticated endpoint.
+* [Accepting client certificates via x509 scheme](#accepting-client-certificates-via-x509-scheme)
+* [Accepting z/OSMF LTPA token](#accepting-zosmf-ltpa-token)
+* [Forwarding x509 client certificate](#forwarding-x509-client-certificate)
```

Comment:
perhaps I am wrong but I thought that client certificate forwarding meant that you would not have an SSO experience.  

Here is the reference:

https://docs.zowe.org/stable/extend/extend-apiml/api-medation-sso-integration-extenders/#accepting-client-certificates-via-x509-scheme

Reply:
I also note that this sentence which is currently in the doc appears to be absent:

> Asking for details about the x509 certificate does not properly participate in SSO as this method cannot accept all authentication methods that are supported upstream of API Mediation Layer.

Perhaps it is no longer true